src/luascript.cpp

```diff
@@ -9159,9 +9162,37 @@ int LuaScriptInterface::luaPlayerSetBankBalance(lua_State* L)
 	return 1;
 }
 
+int LuaScriptInterface::luaPlayerHasStorage(lua_State* L)
+{
+	// player:hasStorage(key)
+	Player* player = getUserdata<Player>(L, 1);
+	if (!player) {
+		lua_pushnil(L);
+		return 1;
+	}
+
+	lua_pushboolean(L, player->storageMap.find(getNumber<uint32_t>(L, 2)) != player->storageMap.end());
+	return 1;
+}
+
+int LuaScriptInterface::luaPlayerClearStorage(lua_State* L)
+{
+	// player:clearStorage(key)
+	Player* player = getUserdata<Player>(L, 1);
+	if (!player) {
+		lua_pushnil(L);
+		return 1;
+	}
+
+	player->addStorageValue(getNumber<uint32_t>(L, 2), -1, false);
+
+	lua_pushboolean(L, true);
+	return 1;
+}
+
 int LuaScriptInterface::luaPlayerGetStorageValue(lua_State* L)
 {
-	// player:getStorageValue(key)
+	// player:getStorageValue(key[, default = -1])
```

Comment:
`default` -> `defaultValue`

Reply:
use the github web api, insert suggestion